notebooks/corridor/corridor_bucharest.qmd

```diff
@@ -22,7 +22,7 @@ In this notebooks we explore how to delineate river corridor edges using Buchare
 
 ```{r}
 # define a directory to store some outputs
-output_dir = "."
+output_dir = "data/generated"
```

Comment:
Note that this does not work in an RStudio project. Relative paths might be handled differently in VS Code. I fixed this with `here::here("data/generated")`. Does this work in VS Code too? If yes, I would prefer this in notebooks.

Reply:
Yes - absolute (project) paths works in VS Code as well, I think it was Sarah's setup that for some reason required her to set relative paths.